css/style.css

```diff
@@ -0,0 +1,557 @@
+  flex: 1.7;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.menu__input {
+  height: 30px;
+  border-top: 0;
+  border-right: 0;
+  border-left: 0;
+  border-bottom: 1px solid #000;
+  position: relative;
+}
+
+.menu__button {
+  position: absolute;
+  background: none;
+  border: 0;
+  align-self: center;
+}
+
+.menu__button:hover {
+  border: 1px solid #000;
+}
+
+.menu__item_cart {
+  font-size: 10px;
+  color: #fff;
+  background: #000;
+  padding: 2px 5px;
+  margin: 0px 3px;
+  border-radius: 50%;
+}
+
+.menu__USA{
+  width: 20px;
+  height: 20px;
+}
+
+.header__container {
+  height: 280px;
+  background: url(..//images/rob-potter-422534-unsplash.png) no-repeat;
+  padding-left: 60px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.container__title {
+  font-size: 30px;
+  font-weight: 700;
+  color: #fff;
+  text-transform: uppercase;
+  max-width: 195px;
+  margin-bottom: 5px;
+}
+
+.button {
+  text-transform: uppercase;
+  font-size: 12px;
+  text-decoration: none;
+  cursor: pointer;
+  font-weight: 700;
+}
+
+.button:hover {
+  text-decoration: underline;
+}
+
+.container__button {
+  color: #fff;
+}
+
+.products {
+  padding-bottom: 45px;
+}
+
+.title {
+  font-size: 22px;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+.products__title {
+  text-align: center;
+  padding: 32px 0px 0px 0px;
+}
+
+.products__items {
+  display: flex;
+  justify-content: space-around;
+}
+
+.sneakers {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 261px;
+}
+
+.sneakers__image {
+  width: 188px;
+  height: 173px;
+  padding-bottom: 30px;
+}
+
+.sneakers__name {
+  font-size: 12px;
+  margin: 0;
+}
+
+.sneakers__description {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex: 1;
+}
+
+.sneakers__description_price {
+  font-size: 14px;
+  font-weight: 900;
+  margin: 5px 0;
+  flex: 3;
+}
+
+.sneakers__description_colors {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex: 1;
+}
+
+.color_mobile {
+  display: none;
+}
+
+.products__items_border {
+  margin: 43px 30px;
+  height: 100px;
+  max-width: 1px;
+  flex: 1;
+}
+
+.history {
+  background: #000;
+  height: 494px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+}
+
+.history__video {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-left: 32px;
+  padding-right: 45px;
+}
+
+.history__video_main {
+  position: relative;
+  width: 644px;
+  height: 358px;
+}
+
+.history__video_background {
+  position: absolute;
+}
+
+.history__video_play {
+  position: absolute;
+}
+
+.history__text {
+  width: 266px;
+}
+
+.history__title {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0;
+}
+
+.history__text_main {
+  font-size: 14px;
+}
+
+.history__button {
+  color: #fff;
+}
+
+@media screen and (max-width: 1169px) {
+  body {
+	  max-width: 930px;
+	  margin: 0 auto;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Final-Axion-cite/pullRequest?prid=5985384)